### DISCUSSION TITLE
Possibility to hide individual entries of list

### DISCUSSION THREAD

**Discussion by andy1463** [1Remote/1Remote#574] [Ideas]
Would be great if there would be an option to hide individual entries in the overview list, if they are not needed at the moment or if some teammembers don't need access to it.

**Comment by VShawn** [comment]
I understand that this is a valuable team feature.

But if want to hide specific data for a particular person, then it would be need to add an account-based authorization module. However, I currently have no plans and not much available time to add such a module to 1Remote.

**Reply by VShawn** [reply]
I suddenly realized that my previous thinking was not enough. not only would it require to add an account module, but also need developing a new server-side program to validate account and distribute server configurations. Additionally, there would be a need for a web UI to configure remote access for each account. 

workload increase lol.

**Comment by majkinetor** [comment]
Tags are literary used for that. If you for example create a tag `noncritical` you can filter it out using negative tag filter `-#noncritical`. Conversely, you can add for example `main` tag to everything that you want always visible and pin that tag.

**Reply by majkinetor** [reply]
It might be somewhat problematic that individual members can't use private tags and must rely on remote tags. 

But, I guess private tags are way simpler to implement eventually than introducing way way more problematic feature such as detailed user rights management (which at this moment of 1RM phase is most definitelly overkill).
